aldryn_newsblog/boilerplates/bootstrap3/templates/aldryn_newsblog/article_detail.html

```diff
@@ -4,6 +4,13 @@
 {% block title %}{{ article.title }} - {{ block.super }}{% endblock %}
 {% block breadcrumb %}{% endblock %}
 
+{% block html_tag %} itemscope itemtype="http://schema.org/Blog" {% endblock %}
+
+{% block meta_tags %}
+    <meta name="viewport" content="width=device-width,initial-scale=1">
```

Comment:
meta_tags is very generic here, that can break your frontend. Should be something like {% blog_meta_tags %} or so. Also the viewport meta tag should be handled by the project frontend, not the blog /cc @vxsx